src/ad-hoc-visualizations/needs-review/visualization.tsx

```diff
@@ -56,7 +61,7 @@ export const NeedsReviewAdHocVisualization: VisualizationConfiguration = {
     getIdentifier: () => AdHocTestkeys.NeedsReview,
     visualizationInstanceProcessor: () => VisualizationInstanceProcessor.nullProcessor,
     getNotificationMessage: (selectorMap, key, warnings) =>
-        getNotificationMessage(selectorMap, warnings),
+        getNotificationMessage(selectorMap, warnings), // not using this
```

Comment:
I'm not certain what the comment means? Why are we not using the parameter? Is it possible to pass in a value that indicates the parameter is not meant to be used?

Reply:
This was also a note-to-self comment, about how we are now using the unified result sender to send notifications for needs review now instead of this.  Removed.